compute-nodes.yml

```diff
@@ -12,7 +12,7 @@
         peerdns: '"no"'
         master: bond0
         slave: '"yes"'
-        mtu: 9000
+        mtu: "{{mtu | default (8964) }}"
```

Comment:
Should the default be something standard like 1500 or 9000 and not something that will just work in our environment?